README.md

```diff
@@ -22,49 +34,149 @@ You can do it manually following the instructions from [rust-lang.org](https://w
+4. Run yor miner with ` --repo <node working directory> --miner-repo <miner work directory> --owner <your public address> --miner-api <node api open port> --sector-size <sector size>`
+
+## Advanced Building
+This aproach allows to build our project with your modifications,
+but be sure that your realisation follows consensus rules.
+
+1. clone [funon-filecoin project](https://github.com/filecoin-project/cpp-filecoin "cpp-filecoin project")
+
+2. go to `./cpp-filecoin` directory
+
+3. run `$git submodule --recursive --update`
+
+4. build targets: node - for node executable, miner_main- for miner executable.
+
+5. First build will likely take long time. However, you can cache binaries to [hunter-binary-cache]. It is suggested to build project with clang-tidy checks, however if you wish to omit clang-tidy step, you can use `cmake ..` instead.
+
+- Tests can be run with:
+  ```
+  cd build
+  ctest
+  ```
+
+6. node executable can be found at: `./<build_repository>/core/node/main`.
+7. miner executable can be found at:
+   `./<build_repository>/core/miner/main`
+8. next steps are similar to common miner and node establishing which can be found at **Fuhon node establishing** and **Fuhon miner establishing**
+
+9. To finish your building you may also want to add your github token, In such case you need to set up two environment variables:
 ```
 GITHUB_HUNTER_USERNAME=<github account name>
 GITHUB_HUNTER_TOKEN=<github token>
 ```
-To generate github token follow the [instructions](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line). Make sure `read:packages` and `write:packages` permissions are granted (step 7 in instructions).
+To generate github token follow the [instructions](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line). Make sure `read:packages` and `write:packages` permissions are granted.
 
-This project is can be built with
 
-```
-mkdir build && cd build
-cmake -DCLANG_TIDY=ON ..
-make -j
-```
 
-It is suggested to build project with clang-tidy checks, however if you wish to omit clang-tidy step, you can use `cmake ..` instead.
+## Configuration Lotus-Fuhon
+### Lotus-node and Fuhon-miner
+1. To start up a lotus node at first you need to get [lotus executable](https://github.com/filecoin-project/lotus/releases/ "lotus releases")
+2. You can run your lotus node with the next command: `$lotus --repo <node working directory> daemon --genesis genesis.car --import-key PRESEAL/pre-seal-MINER.key --profile bootstrapper`, more about running lotus software can be found in [filecoin-docs](https://docs.filecoin.io/get-started/lotus/installation "lotus filecoin documentation").
```

Comment:
"preseal" is only applicable to local devnet, not mainnet/interopnet

Reply:
fixed